booksRead.js

```diff
@@ -0,0 +1,63 @@
+var books = [
+    {
+      title: "The Design of Everyday Things",
+      author: "Don Norman",
+      alreadyRead: false
+    },
+    {
+      title: "The Most Human Human",
+      author: "Brian Christian",
+      alreadyRead: true
+    },
+    {
+      title: "In Search of Lost Time",
+      author: "Marcel Proust",
+      alreadyRead: true
+    },
+    {
+      title: "Ulysses",
+      author: "James Joyce",
+      alreadyRead: true
+    },
+    {
+      title: "The Great Gatsby",
+      author: "F. Scott Fitzgerald",
+      alreadyRead: true
+    }
+  ];
+
+  //creating h1
+let header = document.createElement('h1');
+
+let divBook = document.querySelector('.books');

```

Comment:
I think you was testing this in other file with different class name.
```suggestion
let divBook = document.querySelector('.favoriteBooks');
```